src/Registration/pReg/Reg.py

```diff
@@ -146,7 +146,10 @@ def get_as_deformation_field(self, ref):
 
 class NiftiImageData(SIRF.ImageData):
     """General class for nifti image data."""
-
+    ADD      = 0
+    SUBTRACT = 1
+    MULTIPLY = 2
+    DIVIDE   = 3
```

Comment:
Python doesn't have private members, but i believe there is a convention to start them with `_` or something like that. As  `help(NiftiImageData)` will show these members, I think it's important to at least convey to users that they shouldn't ever use this